### PR TITLE
Mobile header buttons

### PR DESCRIPTION
**Pull request by siggakogk**
# Updated design of mobile search & menu buttons

[Takki í nav-bar
](https://app.asana.com/0/0/1194749547178208/f)

## What

Update UI of mobile buttons in header
When clicking on search icon the menu should open with the search box focused
When clicking the menu button the menu should open in a normal state
Remove blue background in icon buttons

## Screenshots / Gifs

Screen Recording 2020-10-06 at 12 01 43

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

**Review by stjanilofts — DISMISSED** [review]

File: apps/web/components/Header/ComboButton.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  showSearch,
+}) => {
+  const { t } = useI18n()
+
+  return (
+    <>
+      {showSearch && (
+        <Box
+          component={ReaButton}
+          as={'button'}
```
Comment (barabrian) [inline apps/web/components/Header/ComboButton.tsx:28]: this should be default, probably can remove this line

File: apps/web/components/Header/ComboButton.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+          className={cn(styles.buttonBase, styles.searchButton)}
+          onClick={sideMenuSearchFocus}
+        >
+          <Icon type={'search'} width={15} color="blue400" />
+        </Box>
+      )}
+      <Box
+        component={ReaButton}
+        as={'button'}
```
Comment (barabrian) [inline apps/web/components/Header/ComboButton.tsx:37]: this should be default, probably can remove this line

**Review by barabrian — COMMENTED** [review]
LGTM, minor comments

**Review by barabrian — DISMISSED** [review]
LGTM

**Review by jeremybarbet — DISMISSED** [review]
LGTM

**Review by barabrian — APPROVED** [review]

**Pull request merged**